src/components/com_tjcertificate/site/views/certificates/tmpl/my.php

```diff
@@ -167,7 +165,7 @@
 										}
 										else
 										{
-											echo ($data[0]->title ? $data[0]->title : "-");
+											echo (($data && $data[0] && $data[0]->title) ? $data[0]->title : "-");
```

Comment:
@shindebalu Can we use isset() method to check whether the attribute is set on the object or not?

echo isset($data[0]->title) ? $data[0]->title : "-";